docker/conf/log.config

```diff
@@ -49,17 +49,35 @@ handlers:
     class: logging.StreamHandler
     formatter: precise
 
-{% if not SYNAPSE_LOG_SENSITIVE %}
-{#
-  If SYNAPSE_LOG_SENSITIVE is unset, then override synapse.storage.SQL to INFO
-  so that DEBUG entries (containing sensitive information) are not emitted.
-#}
 loggers:
+    # This is just here so we can leave `loggers` in the config regardless of whether
+    # we configure other loggers below (avoid empty yaml dict error).
+    _placeholder:
+        level: "INFO"
+
+    {% if not SYNAPSE_LOG_SENSITIVE %}
```

Comment:
Should we remove `SYNAPSE_LOG_SENSITIVE` in favor of `SYNAPSE_LOG_TESTING` here?

The docs around `SYNAPSE_LOG_SENSITIVE` already mention it's for "a developer attempting to debug something
  particularly tricky."